subset/bacnet/picstest/Dockerfile.test_bacpics

```diff
@@ -0,0 +1,22 @@
+FROM daq/aardvark:latest as bacnet_build
```

Comment:
Your dockerfiles are all nearly identical -- should all be the same thing.  This is the same problem that Ed was having with some of the other tests, so we can walk through it this afternoon together and clear up the right solution.